deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -563,3 +573,11 @@ def check_agent_received_message(message_queue, search_pattern, timeout=5, updat
 
     queue_monitor.start(timeout=timeout, callback=monitoring.make_callback(search_pattern, '.*'),
                         update_position=update_position)
+
+
+def escapee_callback_message(message):
+    escaped = message.translate(str.maketrans({"]":  r"\]",
+                                              "[":  r"\[",
+                                              "(":  r"\(",
+                                              ")":  r"\)"}))
+    return escaped
```

Comment:
You don't need to do this. You can actually simplify it by using `re.escape(str)`.

Reply:
Done in 20cadf88fd2d3f4777f24434f6ca2b8e95ff0f11